docs/sources/clients/promtail/stages/geoip.md

```diff
@@ -0,0 +1,129 @@
+---
+title: geoip
+description: geoip stage
+---
+# geoip
+
+The `geoip` stage is a parsing stage that reads a ip address and 
+populates the labelset with geoip fields. Maxmind's GeoIP2 databse is used for the lookup.
+
+Populated fields for City db:
```

Comment:
Question, in line 8 it says this is using "GeoIP2 database", and in line 10 it says "City database".    Which is correct?  Could we clarify or standardize the terminology used for the database?

Reply:
The general term used is `GeoIP2 database` but is has many types of DB's including `City`, `ASN` and `Country`.  Lines `10` and `23` indicates the same with corresponding documentation.